model/riscv_insts_vext_utils.sail

```diff
@@ -560,6 +560,15 @@ function f_is_SNaN(xf) = {
   }
 }
 
+val f_is_at_least_one_SNaN : forall 'm, 'm in {16, 32, 64}. (bits('m), bits('m)) -> bool
+function f_is_at_least_one_SNaN (op1, op2) = {
+  match 'm {
+    16 => f_is_SNaN_H(op1) | f_is_SNaN_H(op2),
```

Comment:
An alternative way to do this that is simpler is:

```
overload f_is_SNaN = { f_is_SNaN_H, f_is_SNaN_S, f_is_SNaND }

...
    if f_is_SNaN (op1) | f_is_SNaN(op2) then {
```

You can do that for the other things too like `riscv_softfloat_enter/leave`, `riscv_softfloat_eq`.

Reply:
Good to know overload, will update it.